charts/kong/values.yaml

```diff
@@ -578,6 +578,8 @@ enterprise:
     smtp_auth_type: nil
     smtp_ssl: nil
     smtp_starttls: true
+    smtp_mock: on
```

Comment:
Please quote `on` (so make it "on"). YAML's boolean interpretation set is large and and this will be interpreted as a boolean `true` instead of a string `on`.
https://yaml.org/type/bool.html